ui/src/app/common/services/orb.service.ts

```diff
@@ -78,7 +63,20 @@ export class OrbService implements OnDestroy {
   }
 
   observe<T>(observable: Observable<T>) {
-    return this.poller$.pipe(
+    const controller = merge(
```

Comment:
Reusing this controller instance to all observed things was causing issues.
Now, observed things stop being observed after component unsubscribes its 'async' properties.